d2bs/kolbot/libs/common/Misc.js

```diff
@@ -565,11 +569,48 @@ var Item = {
 		};
 	},
 
+        getBodyLocMerc: function (item) {
+		var bodyLocMerc;
+
+		switch (item.itemType) {
+		case 2: // Shield
+			bodyLocMerc = 5;
+
+			break;
+		case 3: // Armor
+			bodyLocMerc = 3;
+
+			break;
+
+		case 37: // Helm
+		case 75: // Circlet
+			bodyLocMerc = 1;
+
+			break;
+		case 27: //
+		case 30: //
+		case 33: //
+		case 34: //
+			bodyLocMerc = 4;
+
+			break;
+		default:
+			return false;
+		}
+
+		if (typeof bodyLocMerc === "number") {
```

Comment:
refactor idea, remove this "number" check and assign as array to begin with:
bodyLocMerc = [5]; etc

Reply:
This is need it as its need it for player auto equip. Im wrong?

Reply:
break;
		default:
			return false;
		}

		if (typeof bodyLoc === "number") {
			bodyLoc = [bodyLoc];
		}

		return bodyLoc;
	},